test/src/IO/WorldReaderTest.cpp

```diff
@@ -693,6 +693,50 @@ namespace TrenchBroom {
             
             delete world;
         }
+        
+        TEST(WorldReaderTest, parseAttributeWithEscapedPathAndTrailingBackslash) {
+            const String data("{"
+                              "\"classname\" \"worldspawn\""
+                              "\"path\" \"c:\\\\a\\\\b\\\\c\\\\\""
+                              "}");
```

Comment:
I had used raw string literals, only to have VC++ being unable to parse them. So they are banned, sorry.